### PR TITLE
changed common goals, styled toast

### PR DESCRIPTION
**Pull request by n-lina**

### CODE REVIEW

File: frontend/android/app/src/main/java/com/dailydash/MainApplication.java [hunk 1]
```diff
@@ -4,6 +4,8 @@
 import android.content.Context;
 import com.facebook.react.PackageList;
 import com.facebook.react.ReactApplication;
+import cl.json.RNSharePackage;
```
Comment (jeremywiens) [inline frontend/android/app/src/main/java/com/dailydash/MainApplication.java:7]: Could you remove these? They are causing codacy errors

File: frontend/app/models/user-store/user-store.ts [hunk 2]
```diff
@@ -20,13 +20,13 @@ const toastAndroid = (message: string) => {
   ToastAndroid.showWithGravity(
     message,
     ToastAndroid.LONG,
-    ToastAndroid.TOP,
+    ToastAndroid.CENTER,
```
Comment (jeremywiens) [inline frontend/app/models/user-store/user-store.ts:23]: The only concern I have with the toastr in the center is if they make the tests fail, like how the warning made the tests not able to press the buttons.

Have you tried that?

Comment (n-lina) [reply frontend/app/models/user-store/user-store.ts:23]: oh true, ill change it back to top

File: frontend/android/app/src/main/java/com/dailydash/MainApplication.java (hunk 1 above)
Comment (n-lina) [reply frontend/android/app/src/main/java/com/dailydash/MainApplication.java]: done

**Review by jeremywiens — APPROVED** [review]

**Pull request merged**